tests/automation/utilities/utils.ts

```diff
@@ -295,60 +295,42 @@ export async function hasElementBeenDeleted(
   window: Page,
   strategy: Strategy,
   selector: string,
-  maxWait?: number
-) {
-  const fakeError = `Element ${selector} has been found... oops`;
+  text?: string,
+  maxWait?: number,
+): Promise<boolean> {
   try {
-    await waitForElement(window, strategy, selector, maxWait);
-    throw new Error(fakeError);
-  } catch (e) {
-    if (e.message === fakeError) {
-      throw e;
-    }
-  }
-  console.info(`${selector} has not been found, congrats`);
-}
+    await waitForElement(window, strategy, selector, maxWait, text);
```

Comment:
if you catch all the exceptions and don't throw anything new, your test will pass, which in this case you don't want